### PR TITLE
Adding support for the clientKey and upgrading the adyen.js file to v3.15.1

### PR DESCRIPTION
**Pull request by bryancresswell**
Added support for the new `clientKey` as well as upgrading the `adyen.js` and `adyen.css` files to `v3.15.1`  on all examples, allowing for support of new features such as selecting the card brand for dual-branded cards.

The `clientKey` was added with a mock endpoint on the server which would take the `CLIENT_KEY` env variable from the `.env` file as there is currently no endpoint for the `clientKey` in the meantime. 

Also briefly updated the README with some extra troubleshooting tips in the event that the forms do not load due to invalid `clientKey` or if the origins have yet to be whitelisted on the CA.

### CODE REVIEW

File: src/ach/ach.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-// 0. Get originKey
-getOriginKey().then(originKey => {
+// 0. Get clientKey
+getClientKey().then(clientKey => {
     // 1. Create an instance of AdyenCheckout
     const checkout = new AdyenCheckout({
         environment: 'test',
-        originKey: originKey // Mandatory. originKey from Customer Area
+        clientKey: clientKey // Mandatory. clientKey from Customer Area
```
Comment (pabloai) [inline src/ach/ach.js:6]: This is the same as just doing this:
```suggestion
        clientKey // Mandatory. clientKey from Customer Area
```
But maybe it's more clear to leave it as it is. What do you think?

Comment (bryancresswell) [reply src/ach/ach.js:6]: I think yes, we can leave it like that, but since this is an intro, maybe we can put it clearer as `clientKey: clientKey`.

Comment (bryancresswell) [reply src/ach/ach.js:6]: Hi @pabloai, I've also modified the resources to load from `checkoutshopper-test` instead of `checkoutshopper-live`!

**Review by pabloai — APPROVED** [review]

**Comment by pabloai** [comment]
Great work @bryancresswell !

**Pull request merged**